examples/config/config.properties

```diff
@@ -0,0 +1,267 @@
+name = "SR3"
+
+
+[[emme.highway.classes]]
+name = "trk"
+description = "truck"
+emme_mode = "t"
+excluded_links = [ "is_toll_trk", "is_sr2",]
+value_of_time = 37.87
+operating_cost = 31.28
+toll = "@bridgetoll_sml"
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_vsm", "bridgetoll_sml", "bridgetoll_med",]
+[[emme.highway.classes.demand]]
+file = "commercial"
+name = "VSTRUCK"
+
+[[emme.highway.classes.demand]]
+file = "commercial"
+name = "STRUCK"
+
+[[emme.highway.classes.demand]]
+file = "commercial"
+name = "MTRUCK"
+
+
+[[emme.highway.classes]]
+name = "lrgtrk"
+description = "large truck"
+emme_mode = "l"
+excluded_links = [ "is_toll_lrgtrk", "is_auto_only",]
+value_of_time = 37.87
+operating_cost = 31.28
+toll = "@bridgetoll_lrg"
+passenger_car_equivalent = 2.0
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_lrg",]
+[[emme.highway.classes.demand]]
+file = "commercial"
+name = "CTRUCK"
+factor = 2.0
+
+
+[[emme.highway.classes]]
+name = "datoll"
+description = "drive alone toll"
+emme_mode = "D"
+excluded_links = [ "is_sr2",]
+value_of_time = 18.93
+operating_cost = 17.23
+toll = "@toll_da"
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_da", "valuetoll_da", "tolldist",]
+
+[[emme.highway.classes]]
+name = "sr2toll"
+description = "shared ride 2 toll"
+emme_mode = "E"
+excluded_links = [ "is_sr3",]
+value_of_time = 18.93
+operating_cost = 17.23
+toll = "@toll_sr2"
+toll_factor = 0.5714285714285714
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_sr2", "valuetoll_sr2", "hovdist", "tolldist",]
+[[emme.highway.classes.demand]]
+file = "household"
+name = "SR2_PAY_{period}"
+factor = 0.5714285714285714
+
+[[emme.highway.classes.demand]]
+file = "air_passenger"
+name = "SR2TOLL"
+
+[[emme.highway.classes.demand]]
+file = "internal_external"
+name = "SR2TOLL"
+
+
+[[emme.highway.classes]]
+name = "sr3toll"
+description = "shared ride 3+ toll"
+emme_mode = "F"
+excluded_links = []
+value_of_time = 18.93
+operating_cost = 17.23
+toll = "@toll_sr3"
+toll_factor = 0.4
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_sr3", "valuetoll_sr3", "hovdist", "tolldist",]
+[[emme.highway.classes.demand]]
+file = "household"
+name = "SR3_PAY_{period}"
+factor = 0.4
+
+[[emme.highway.classes.demand]]
+file = "air_passenger"
+name = "SR3TOLL"
+
+[[emme.highway.classes.demand]]
+file = "internal_external"
+name = "SR3TOLL"
+
+
+[[emme.highway.classes]]
+name = "trktoll"
+description = "truck toll"
+emme_mode = "T"
+excluded_links = [ "is_sr2",]
+value_of_time = 37.87
+operating_cost = 31.28
+toll = "@toll_sml"
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_vsm", "bridgetoll_sml", "bridgetoll_med", "valuetoll_vsm", "valuetoll_sml", "valuetoll_med",]
+[[emme.highway.classes.demand]]
+file = "commercial"
+name = "VSTRUCKTOLL"
+
+[[emme.highway.classes.demand]]
+file = "commercial"
+name = "STRUCKTOLL"
+
+[[emme.highway.classes.demand]]
+file = "commercial"
+name = "MTRUCKTOLL"
+
+
+[[emme.highway.classes]]
+name = "lrgtrktoll"
```

Comment:
Is there a space limit on these highway class name? if not, lets spell things out

Reply:
Yes, limited to max 8 character limit for mode name in Emme, and can bump up against the 40 char limit for matrices along with the period and skim name (not an issue so far). The "description" is supposed to provide a longer, clearer name.